sdk/anomalydetector/ai-anomaly-detector-rest/samples-dev/sample_multivariate_detection.ts

```diff
@@ -69,47 +69,39 @@ export async function main() {
 
   // train model
   const createModelResult = await client
-    .path("/{ApiVersion}/multivariate/models")
-    .post(createMultivariateModelParameters)
-    .then((res) => {
-      if (!("modelId" in res.body)) {
-        throw res.body;
-      }
-      return res.body;
-    });
+    .path("/multivariate/models")
+    .post(createMultivariateModelParameters);
+
+  if (isUnexpected(createModelResult)) {
```

Comment:
Updated isUnexpected, @saintzou66  can you please verify if this is now working?

Reply:
It seems isUnexpected is still not working.
Here is the logging:
pathParts:
 [ '', 'anomalydetector', 'v1.1', 'multivariate', 'models' ]
candidateParts:
[ '', 'timeseries', 'entire', 'detect' ]
[ '', 'timeseries', 'last', 'detect' ]
[ '', 'timeseries', 'changepoint', 'detect' ]
[ '', 'multivariate', 'models' ]
[ '', 'multivariate', 'models', '{modelId}:detect-batch' ]
[ '', 'multivariate', 'models', '{modelId}:detect-last' ]

As they have different parts of the whole url, the geParametrizedPathSuccess function will always return am empty url

Reply:
There is a bug when do comparation. We should ignore the first empty string.